src/components/savedArticles/SavedArticles.js

```diff
@@ -52,8 +52,8 @@ export default function SavedArticles(props) {
 			<h2 className='saved-articles__subtitle'>{currentUser.username}, you have {savedArticles.length} saved articles</h2>
 			<p className='saved-articles__by'>By keywords: <span className='saved-articles__by_keywords'>{checkAuthors()}</span></p>
 			<ul className={`saved-articles__list${isSavedArticles ? '' : '_closed'}`}>
-				{savedArticles.map((article) =>
-					<Article gettingSavedArticles={gettingSavedArticles} isHomePage={false} key={generateKey(article)} article={article} />
+				{savedArticles.map((article, index) =>
+					<Article gettingSavedArticles={gettingSavedArticles} key={index} isHomePage={false} article={article} />
```

Comment:
` key={index} `

It’s better to use `_id` of the card instead of the `index` for attribute `key`

Here is why we shouldn’t use `index` for that: [click here]( https://programmingwithmosh.com/react/why-do-i-need-keys-in-react-lists/#:~:text=React%20recommends%20that%20you%20do,to%20some%20unstable%20component%20behaviour.)

Reply:
it's better to correct this